scipion/__main__.py

```diff
@@ -193,6 +193,9 @@ class Vars:
     SCIPION_PYTHON = PYTHON
     SCIPION_TESTS_CMD = os.environ.get("SCIPION_TESTS_CMD", '%s %s' % (SCIPION_EP, MODE_TESTS))
 
+    # Priority package list
+    SCIPION_PRIORITY_PACKAGE_LIST = "pwem tomo pwchem"
```

Comment:
what is pwchem?

Reply:
A whole set of plugins for ligands and interactions:  https://github.com/scipion-chem